openquake/commonlib/readinput.py

```diff
@@ -665,9 +670,14 @@ def get_exposure_lazy(fname):
         area = conversions.area
     except NameError:
         area = LiteralNode('area', dict(type=''))
+    cts = [ct for ct in conversions.costTypes if ct['name'] in ok_cost_types]
+    cost_types = [(ct['name'], ct['type'], ct['unit']) for ct in cts]
```

Comment:
I guess this could be done in one line instead of 2, also because you are not using `cts` anywhere else